README.md

```diff
@@ -1,2 +1,22 @@
-# Words
-Example for WSU CEG-6110/4110 Spring 2022
+# WordsExampleSpring2022
+
+@mainpage Project
+This entire project is part of a demonstration for Wright State University (WSU) CSE-6110/4110 "Intro. Software Eng." Spring 2022.
+
+## Overview
+
+Software Engineering is the application of engineering processes to create software specifications, designs, implementations, tests, and sustainment. This project shows one concrete application of a process to document the design of software. The goal is to produce useful maintainable software design documentation with a minimum of extra effort beyond the effort that would be expended on designing and implementing software. In other words, the goal is to minimize any extra effort needed document the design that happens whether it is documented or not.
+
+## Doxygen
+The basic approach is to use the free open-source cross-platform multi-language Doxygen program to extract formatted comments from software "stubs". Stubs are an outline of software that has not yet been implemented. Create the stubs early in the design process and reuse them as part of software implementation. We don't want any wasted effort!
+
+## Traceability 
+One of the key features of a design is traceability to requirements (or User Stories). How do you know when you are done designing? You are done when the design describes HOW (in a general sense) each of the requirements will be met by software. The actual implementation of software ultimately describes HOW requirements are met in the most specific way possible, and implementations need to trace back to design and requirements too, but little or no implementation is needed during design. To trace design to requirements, Doxygen @anchore and custom Doxygen @req tags are used in formatted comments surrounding the stubs. Together, the tags provide traceability from design to requirements and from requirements to design. Traceability from design to Requirements can succinctly answer the question, "Why does this deigned element exist?"It exists to fully or partially meet referenced requirements." Doxygen makes convenient hyperlinks so clicking on a requirement referenced in a design takes the reader directly to the text of the referenced requirement.
```

Comment:
In Doxygen, I believe the tag is "@anchor", rather than "@anchore". Fix spelling.